dev-support/bin/create-release

```diff
@@ -441,9 +441,9 @@ function dockermode
     echo "LABEL org.apache.hadoop.create-release=\"cr-${RANDOM}\""
 
     # setup ownerships, etc
-    echo "RUN groupadd --non-unique -g ${group_id} ${user_name}"
-    echo "RUN useradd -g ${group_id} -u ${user_id} -m ${user_name}"
-    echo "RUN chown -R ${user_name} /home/${user_name}"
+    echo "RUN groupadd --non-unique -g ${group_id} ${user_name}; exit 0;"
```

Comment:
Another same PR about this changes?

Reply:
@Hexiaoqiao Thank you for reviewing the code!
 We have improved the hadoop script at https://github.com/apache/hadoop/pull/6448. I think hadoop-thirdparty should also be improved.